src/ccl_core.c

```diff
@@ -533,7 +547,7 @@ ccl_parameters ccl_parameters_create(
 	  params.m_nu[0] = 0.;
   }
   // Free mnu_in
-  if (mnu_in != NULL) free(mnu_in);
+  if (mnu_in != NULL) free(mnu_in);*/
```

Comment:
Please remove old commented code.

Reply:
you must have forgotten to push a file or something. The code is still in the diff.